src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -8,14 +8,15 @@ import {addDays, periodSinceEpoch, minutesBetween, getCurrentDate, daysBetweenUT
 import {I18n} from 'locale';
 import {Observable, MapObservable} from 'shared/Observable';
 import {captureException, captureMessage} from 'shared/log';
+import {ACCESSIBLE} from 'react-native-secure-key-store';
 
 import {BackendInterface, SubmissionKeySet} from '../BackendService';
 
 import exposureConfigurationDefault from './ExposureConfigurationDefault.json';
 import exposureConfigurationSchema from './ExposureConfigurationSchema.json';
 import {ExposureConfigurationValidator, ExposureConfigurationValidationError} from './ExposureConfigurationValidator';
 
-const SUBMISSION_AUTH_KEYS = 'submissionAuthKeys';
+const SUBMISSION_AUTH_KEYS = 'submissionAuthKey';
```

Comment:
Assuming this is just cosmetic, but figured I'd ask just in case this would potentially cause any issues elsewhere.

Reply:
Not cosmetic... It was changed so that it wouldn't conflict with any devices that already have the app installed, as changing the accessible options on an existing key doesn't work.

Reply:
I understand this part but let's add version instead of removing `s`. Ex: 

```ts
const SUBMISSION_AUTH_KEYS = 'submissionAuthKeys_1';
```

Reply:
Sounds good let's make that change
```
const SUBMISSION_AUTH_KEYS = 'submissionAuthKeys_1';
```